.semaphore/project.yml

```diff
@@ -1,42 +0,0 @@
-# This file is managed by ServiceBot plugin - Semaphore. The content in this file is created using a common
-# template and configurations in service.yml.
-# Modifications in this file will be overwritten by generated content in the nightly run.
-# For more information, please refer to the page:
-# https://confluentinc.atlassian.net/wiki/spaces/Foundations/pages/2871296194/Add+SemaphoreCI
-apiVersion: v1alpha
-kind: Project
-metadata:
-  name: terraform-provider-confluent-internal
```

Comment:
It's a bit concerning to see that this file was completely removed, but its name is outdated anyway, and there's this note: "This file is managed by the ServiceBot plugin." anyway.